tests/testcases/test_presets.py

```diff
@@ -172,7 +172,7 @@ def test_find_newest_preset_1(self):
         path = os.path.join(PRESETS, "Bar Device", "picture.png")
         os.mknod(path)
 
-        os.system("find /tmp/input-remapper-test/")
+        os.system("find " + tmp + "/")
```

Comment:
Formatstrings instead of string concatenation please.

```suggestion
        os.system(f"find {tmp}/")
```